ChangeLog.d/pkg-config-files-addition.txt

```diff
@@ -0,0 +1,4 @@
+Features
```

Comment:
We're announcing a new feature, but this isn't tested. Are you planning a follow-up with a test (either expanding an existing cmake-related component or a new component in `tests/scripts/all.sh`)?

Reply:
> We're announcing a new feature, but this isn't tested. Are you planning a follow-up with a test (either expanding an existing cmake-related component or a new component in `tests/scripts/all.sh`)?

Yeah the goal is to add this to automated tests doing a make install and then running `pkg-config`, I have to look at how the test suite is constructed to see where the best place is.

Reply:
See PR for tests: https://github.com/Mbed-TLS/mbedtls/pull/8988